example_allocators/bitmap_allocator/README.md

```diff
@@ -0,0 +1,10 @@
+Simple bitmap allocator implementation, to allocate
```

Comment:
This file needs reformatting to consistent line widths (e.g. 80/100 chars).

Reply:
Fixed in `7906aad`.